app/services/better_shell_adapter.rb

```diff
@@ -0,0 +1,44 @@
+require 'open3'
```

Comment:
Don't think this is required either. Specs still pass without it (can we trust the specs?)

Reply:
i've removed this